src/resources/basis.js

```diff
@@ -479,7 +480,9 @@ function basisDownloadFromConfig(callback) {
             // #ifdef DEBUG
             console.warn("WARNING: unable to load basis wasm module - no config was specified");
             // #endif
+            return false;
         }
+        return true;
     }
```

Comment:
Sorry, the return needs to be outside:
```suggestion
    }
    return true;
```